tasks/section5.yml

```diff
@@ -151,15 +188,77 @@
       - rule_5.1.8
 
 - name: "SCORED | 5.1.8 | PATCH | Ensure at/cron is restricted to authorized users"
-  command: /bin/true
+  file:
+      dest=/etc/at.deny
+      state=absent
```

Comment:
This task needs to be in YAML literal form per https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md